packages/block-editor/src/components/block-parent-selector/index.js

```diff
@@ -74,7 +74,7 @@ export default function BlockParentSelector() {
 				onClick={ () => selectBlock( firstParentClientId ) }
 				label={ sprintf(
 					/* translators: %s: Name of the block's parent. */
-					__( 'Select %s' ),
+					__( 'Select parent %s' ),
```

Comment:
I feel like it should be capitalized: "Select Parent Group". "Select parent Group" looks off to me.

Reply:
😓 I was thinking the opposite. Adding capitalisation here could make me think the candidate entity for selection is "Parent Group" rather than "the parent of this block which is called Group".

Reply:
You're right. Agreed and approved!